tests/foreman/api/test_contentview.py

```diff
@@ -1173,7 +1173,7 @@ def test_negative_readonly_user_actions(
     with pytest.raises(HTTPError):
         target_sat.api.ContentView(server_config=cfg, organization=module_org).create()
     # Check that we can read our content view with custom user
-    content_view = target_sat.api.ContentView(server_config=cfg, id=content_view.id).read()
+    content_view = target_sat.api.ContentView(server_config=cfg, id=content_view.id)
```

Comment:
change this to 
```suggestion
    content_view_read_only_user = target_sat.api.ContentView(server_config=cfg, id=content_view.id)
```
use that across all places where it is required e.g. line number 1195